corehq/apps/data_interfaces/views.py

```diff
@@ -880,7 +880,7 @@ def page_context(self):
         hour = domain_obj.auto_case_update_hour
         context.update({
             'help_site_url': 'https://confluence.dimagi.com/display/commcarepublic/Automatically+Close+Cases',
-            'time': f"{hour}:00" if hour else _('midnight'),
+            'time': f"{hour}:00" if hour else _('midnight'),  # noqa: E999
```

Comment:
I was confused by stickler flagging this earlier - what is it that the linter doesn't like here?

Reply:
I too am confused by it. Daniel mentioned that stickler runs on 3.8, so I tried running this in a 3.8 shell but it works fine there too. I'm assuming it is some combination of the f-string formatting, translation underscore and ternary statement that is throwing stickler off? Would be nice to have a more conclusive answer, but it seems safe to ignore.

Reply:
huh, mysterious, agree it's fine to ignore